app/test_main.py

```diff
@@ -1 +1,40 @@
-# write your code here
+from unittest import mock
```

Comment:
you could actually import `from unittest.mock import MagicMock, patch`
so you would take only what you need